prep.sh

```diff
@@ -32,7 +32,7 @@ for LINE in $LINEZ; do
 	echo '# nothing' >jenkins/split-plugins.txt
 	cp -r jenkins-for-test "megawar-${LINE}"
 	jar uvf megawar-$LINE/WEB-INF/lib/jenkins-core-*.jar jenkins/split-plugins.txt
-	rm -rfv megawar-$LINE/WEB-INF/detached-plugins megawar-$LINE/META-INF/*.{RSA,SF}
+	rm -rfv `# TODO delete all but instance-identity? megawar-$LINE/WEB-INF/detached-plugins` megawar-$LINE/META-INF/*.{RSA,SF}
```

Comment:
FTR this behavior dates to 6c00be1097fcd9e0011991a2f6de5f2095bb5ece because of https://github.com/jenkinsci/plugin-compat-tester/pull/116#issuecomment-509439409 which was three years ago and I barely recall what the issue was.